lib/controllers/auth.js

```diff
@@ -1,80 +0,0 @@
-"use strict";
-
```

Comment:
@ElijahGeorge Are any parts of this file still important to hold onto or are we good with deleting them.

Reply:
👍 I'll add the facebook auth back in